### PR TITLE
[0.9.0]: Adds missing configuration to the hello world demo

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

This PR was initially meant to just add the missing `setup` to the hello world service, but multiple other missing configurations and issues were detected and fixed.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: packages/valory/agents/hello_world/aea-config.yaml [hunk 1]
```diff
@@ -63,9 +64,19 @@ default_connection: null
...
+  benchmark_tool:
+    args:
+      log_dir: ${str:/benchmarks}
   params:
     args:
-      hello_world_message: ${MY_HELLO_WORLD_MESSAGE:str:HELLO_WORLD!}
+      hello_world_message: ${str:HELLO_WORLD!}
+      service_registry_address: null
+      share_tm_config_on_startup: false
+      on_chain_service_id: null
```
Comment (jmoreira-valory) [inline packages/valory/agents/hello_world/aea-config.yaml:75]: Shouldn't these parameters follow the ${<type>:<default_value>} format?

File: packages/valory/services/hello_world/service.yaml [hunk 2]
```diff
@@ -7,43 +7,28 @@ license: Apache-2.0
...
-    benchmark_tool:
-      args: *id001
-3:
-  models:
-    params:
-      args:
-        hello_world_message: ${SERVICE_HELLO_WORLD_HELLO_WORLD_STRING:str:HELLO_WORLD!}
-    benchmark_tool:
-      args: *id001
+models:
```
Comment (jmoreira-valory) [inline packages/valory/services/hello_world/service.yaml:15]: Why this change? The documentation is aligned with having multiple overrides on this file as an example.

Comment (Adamantios) [reply packages/valory/services/hello_world/service.yaml:15]: Ahh I see, I'll revert this. 

However wouldn't it make more sense if the overrides could be different for the agents?

Comment (Adamantios) [reply packages/valory/services/hello_world/service.yaml]: Fixed on e0b8b8d528a7bc73146bdbc7c8e427b7f8cbca78.

File: packages/valory/agents/hello_world/aea-config.yaml (hunk 1 above)
Comment (Adamantios) [reply packages/valory/agents/hello_world/aea-config.yaml]: Fixed on 868df67ff2082675964b0fd6fae3ede68cb17250.

**Review by jmoreira-valory — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]
